oximeter/db/src/lib.rs

```diff
@@ -138,6 +138,12 @@ pub enum Error {
     #[error("Schema update versions must be sequential without gaps")]
     NonSequentialSchemaVersions,
 
+    #[error("Could not read timeseries_to_delete file")]
+    ReadTimeseriesToDelete {
```

Comment:
minor: ReadTimeSeriesToDelete -> ReadTimeSeriesToDeleteFile? This matches with the SqlFile Error in there already.

Reply:
Good idea, done in c9bd27c10